pf9/modules/analytics_utils.py

```diff
@@ -0,0 +1,105 @@
+import analytics
+import uuid
+import math
+import time
+import datetime
+
+# the write_key is the Segment API authorization and identifier, without this no data submission will work. The API Key below is for Dev/Test work
+analytics.write_key = 't1tdSZocQ49Tx3G66lUjJXkOXjUSYRKx'
+
+def on_error(error, items):
+    print('An error occurred:', error)
+
+analytics.debug = True
```

Comment:
Where does this write to?

Reply:
I guess std err, but I removed this and added exception handling to print exceptions in the pf9ctl log file.